waku/v2/node/wakunode2.nim

```diff
@@ -1075,11 +1100,25 @@ when isMainModule:
         discoveryConfig = DiscoveryConfig.init(
           conf.discv5TableIpLimit, conf.discv5BucketIpLimit, conf.discv5BitsPerHop)
 
+      # select dynamic bootstrap nodes that have an ENR containing a udp port
```

Comment:
Maybe brief comment mentioning why this is necessary, i.e. Discovery v5 is only UDP, or link to spec